src/main/java/de/tum/in/www1/artemis/service/iris/websocket/IrisChatWebsocketService.java

```diff
@@ -76,24 +85,38 @@ public static class IrisWebsocketDTO {
 
         private final IrisRateLimitService.IrisRateLimitInformation rateLimitInfo;
 
-        public IrisWebsocketDTO(IrisMessage message, IrisRateLimitService.IrisRateLimitInformation rateLimitInfo) {
+        private final List<PyrisStageDTO> stages;
+
+        public IrisWebsocketDTO(IrisMessage message, IrisRateLimitService.IrisRateLimitInformation rateLimitInfo, List<PyrisStageDTO> stages) {
```

Comment:
this class should be converted into a record as well (could be done in a follow-up)